cmd/approvelist/main.go

```diff
@@ -76,8 +76,8 @@ func main() {
 		}
 		quickfeedStudents[studID] = student
 
-		submissions := courseSubmissions.For(enroll.ID)
-		numApproved := numApproved(enroll.UserID, submissions)
+		enroll.CountApprovedSubmissions(courseSubmissions.For(enroll.GetID()), false)
```

Comment:
Does it matter for approve list if we include the `withActivityDate` information? I think we can drop the second argument to `CountApprovedSubmissions`.